nbbuild/templates/common.xml

```diff
@@ -65,7 +65,7 @@
         <condition property="javahelp.class.path.extension" value="docs/${javahelp.jar.basename}">
             <isset property="has.javahelp"/>
         </condition>
-        <property name="javac.source" value="1.4"/>
+        <property name="javac.source" value="1.6"/>
```

Comment:
I guess I'd suggest to use at least 1.7 as a default. 1.7 is not supported under JDK 12, see:
http://hg.openjdk.java.net/jdk/jdk/rev/999f09bf3464

So using 1.6 would only be a fairly temporary measure.

There are some (minor) differences in type inference between 1.7 and 1.8, so 1.7 is probably safer than 1.8.